### ISSUE TITLE
Error: Cannot find module 'eslint-webpack-plugin'

### ISSUE THREAD

**Issue by poppabear8883** [riasvdv/laravel-mix-eslint#8]
Simply followed the instructions via README and when I run `npm run watch` I receive the error.

**Comment by riasvdv** [comment]
Seems like this plugin isn't compatible with laravel-mix v6, I'll gladly accept a PR if you get it working, otherwise I'll try to do this when I find some time. It's not a high priority for me as I don't use this anymore

**Comment by jasperf** [comment]
That is too bad. Was just doing research on possible setups with eslint webpack plugin and eslint and started thread at https://laracasts.com/discuss/channels/elixir/webpack-eslint-setups-with-laravel-mix-6  . Do think we need something like

```js
.webpackConfig({
    plugins: [
     new ESLintPlugin(options)
      }),
    ],
```

to load plugins in new Laravel Mix 6, but not quite done with the research.

**Comment by dillingham** [comment]
Came to report this also

**Comment by Aslam97** [comment]
You need to install `eslint-webpack-plugin` manually. see https://github.com/riasvdv/laravel-mix-eslint/blob/master/index.js#L9
According to laravel-mix documentation what is returned from `dependencies()` will be installed by mix. i don't know why it doesn't auto install.

> so after you run `npm install laravel-mix-eslint --save-dev` you need to install `eslint-webpack-plugin` run ```npm install eslint-webpack-plugin --save-dev```

**Comment by dillingham** [comment]
@Aslam97 good catch

**Comment by Zhythero** [comment]
Hi it seems that the `eslint` dependency is not auto installed too. Manually ran `yarn add eslint --dev` along with `yarn add eslint-webpack-plugin`.

**Comment by Zhythero** [comment]
So for anyone encountering this issue in the future, do:

`yarn add eslint eslint-webpack-plugin --dev`

or the npm equivalent

**Comment by jasperf** [comment]
Yeah with `eslint-webpack-plugin` added this package does work. Just need to see how I can include prettier in the mix now but ESLint works well. Thanks @Zhythero
